aten/src/ATen/cpu/vec256/vec256_complex_double.h

```diff
@@ -406,32 +406,6 @@ Vec256<c10::complex<double>> inline minimum(const Vec256<c10::complex<double>>&
   return _mm256_or_pd(min, isnan);
 }
 
-template <>
-Vec256<c10::complex<double>> inline clamp(const Vec256<c10::complex<double>>& a, const Vec256<c10::complex<double>>& min, const Vec256<c10::complex<double>>& max) {
```

Comment:
I've removed these, since clamp should not support complex as discussed before. 
If we would like to keep these specialized implementations, then the complex implementation in `vec256_base.h` would have to be changed to match the new clamp behavior, which I've removed now as per your suggestion in the previous PR. I.e. any method declarations in `vec256_complex_double.h` (and equivalent specialized implementations) have to be first defined in `vec256_base.h`. 

Note: When you were reviewing I hadn't removed the complex clamp from `vec256_base.h`, I've missed it, but I've now  fixed it and removed it in the edited PR.

Reply:
Oh I see. I wonder if I was misreading. This is just removing the COMPLEX vectorized clamps, not the FLOAT vectorized clamps, right? That sounds brilliant, thank you.

Reply:
Yes, indeed, this only removes the complex implementation (as indicated by the filename, and the return type of the function) :)